Header header
geometry_msgs/PointStamped center
geometry_msgs/PointStamped[] bounds
float32 visibilityRatio
time measuredTime
